ontology/documentation/ontology.ttl

```diff
@@ -1,165 +1,236 @@
+                            rdfs:subClassOf <http://w3id.org/rml/ExpressionMap> ,
+                                            <http://w3id.org/rml/LogicalIterable> ;
+                            rdfs:comment "A field is an expression map that gives a name to an expression." ;
+                            rdfs:isDefinedBy <http://w3id.org/rml/lv/> ;
+                            rdfs:label "Field" .
 
 
-###  http://w3id.org/rml/LogicalView
-rml:LogicalView rdf:type owl:Class ;
-                rdfs:subClassOf rml:LogicalSource ;
-                rdfs:isDefinedBy <http://w3id.org/rml/lv/> .
+###  http://w3id.org/rml/ForeignKeyAnnotation
+<http://w3id.org/rml/ForeignKeyAnnotation> rdf:type owl:Class ;
+                                           rdfs:subClassOf <http://w3id.org/rml/InclusionDependencyAnnotation> ;
+                                           rdfs:comment "A foreign key annotation is analogous to the notion of foreign key constraint in databases." ;
+                                           rdfs:isDefinedBy <http://w3id.org/rml/lv/> ;
+                                           rdfs:label "Foreign Key Annotation" .
 
 
-###  http://w3id.org/rml/LogicalViewJoin
-rml:LogicalViewJoin rdf:type owl:Class ;
-                    rdfs:isDefinedBy <http://w3id.org/rml/lv/> .
+###  http://w3id.org/rml/IRISafeAnnotation
+<http://w3id.org/rml/IRISafeAnnotation> rdf:type owl:Class ;
+                                        rdfs:subClassOf <http://w3id.org/rml/StructuralAnnotation> ;
+                                        rdfs:comment "A IRI safe annotation indicates that the field content is IRI safe." ;
+                                        rdfs:isDefinedBy <http://w3id.org/rml/lv/> ;
+                                        rdfs:label "IRI Safe Annotation" .
 
 
-###  http://w3id.org/rml/ForeignKeyAnnotation
-rml:ForeignKeyAnnotation rdf:type owl:Class ;
-                      rdfs:subClassOf rml:InclusionDependencyAnnotation ;
-                      rdfs:isDefinedBy <http://w3id.org/rml/lv/> .
+###  http://w3id.org/rml/InclusionDependencyAnnotation
+<http://w3id.org/rml/InclusionDependencyAnnotation> rdf:type owl:Class ;
+                                                    rdfs:subClassOf <http://w3id.org/rml/StructuralAnnotation> ;
+                                                    rdfs:comment "An inclusion dependency annotation is analogous to the notion of inclusion dependency in databases." ;
+                                                    rdfs:isDefinedBy <http://w3id.org/rml/lv/> ;
+                                                    rdfs:label "Inclusion Dependency Annotation" .
 
 
-###  http://w3id.org/rml/IRISafeAnnotation
-rml:IRISafeAnnotation rdf:type owl:Class ;
-                   rdfs:subClassOf rml:StructuralAnnotation ;
-                   rdfs:isDefinedBy <http://w3id.org/rml/lv/> .
+###  http://w3id.org/rml/LogicalIterable
+<http://w3id.org/rml/LogicalIterable> rdf:type owl:Class .
```

Comment:
This could be removed, since it will be imported.

Reply:
It has been added by Widoco. It is not there in the original file, now here available: <https://github.com/kg-construct/rml-lv/blob/main/ontology/documentation/ontology.ttl>